random_menu_generator_gold2.rb

```diff
@@ -0,0 +1,41 @@
+# outputs welcome
+puts "Welcome to Random Menu Generator!"
+print "How many meals would you like to see? "
+how_many = gets.chomp
+
+# outputs error when nil response or response does not include a number
+numbers = [ "0", "1", "2", "3", "4", "5", "6", "7", "8", "9" ]
+unless how_many.length != 0 && numbers.any? { |num| how_many.include?(num)}
+  puts "That doesn't make any sense. Please enter a number."
+end
+
+# assigns array items
+adj1 = [ "chunky", "raunchy", "gelatinous", "gooey", "creamy", "flakey", "molten", "spongy", "corpulent", "animistic", "languid", "greased"]
+adj2 = [ "boozy", "spiced", "creamed", "drunken", "irksome", "steamed", "grilled", "braised", "herbal", "massaged", "moist", "buttered" ]
+food = [ "taco", "cake", "kimchi", "octopus", "pie", "stirfry", "sliders", "cocktail", "turnover", "dumpling", "meatloaf", "waffle" ]
+
+# assigns num initial value of 0
+num = 0
```

Comment:
I would consider declaring this closer to where it is being used as it's not clear what it's for up here.